### PR TITLE
[Merged by Bors] - Implement pause/stop

### PR DESCRIPTION
**Pull request by razvan**
# Description

[CI](https://ci.stackable.tech/job/hbase-operator-it-custom/79/) failed but the failed tests work locally (on kind).

*Please add a description here. This will become the commit message of the merge request later.*

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: Cargo.toml
Comment (Maleware) [inline Cargo.toml:1]: there is a slash too much in https://github.com/stackabletech//operator-rs.git

Comment (sbernauer) [reply Cargo.toml:1]: I remember from the back of my head that it's intentional and needed. You can try to remove it and see how it goes

**Comment by maltesander** [comment]
test https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/hbase-operator-it-custom/80/

File: Cargo.toml
Comment (razvan) [reply Cargo.toml:1]: Hm when I removed the // cargo complained that I have the same origin multiple times (which was true). It only worked with //

**Review by maltesander — APPROVED** [review]
bors merge

**Pull request closed without merging**